assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -13412,6 +13411,10 @@ private void resetAssignment(SessionState state) {
 
         state.removeAttribute(TAG_SELECTOR);
         state.removeAttribute(GB_SELECTOR);
+
+        if (!serverConfigurationService.getBoolean("gradebookng.category.keepLastUsed", true)) {
```

Comment:
is this property name more suitable?
```suggestion
        if (serverConfigurationService.getBoolean("assignment.category.reset", true)) {
```